scripts/apps/authoring-react/fields/dropdown/index.ts

```diff
@@ -26,6 +26,7 @@ export function getDropdownField(): ICustomFieldType<IDropdownValue, IDropdownVa
         getEmptyValue: (config) => config.multiple ? [] : null,
         differenceComponent: Difference,
         configComponent: Config,
+        toOperationalFormat: (valueStorage) => valueStorage,
```

Comment:
if operational and storage formats are the same then you don't need to specify it